public/js/config.js

```diff
@@ -0,0 +1,2 @@
+// REPLACE WITH API KEY
+export const MAPS_API_KEY = "ENTER-API-KEY";
```

Comment:
Add this file to .gitignore so that the credentials aren't visible outside of your project